code/modules/clothing/suits/f13armor.dm

```diff
@@ -1359,8 +1359,36 @@
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS|HANDS
 	slowdown = 0.05
 
+//Fluff Stuff
 /obj/item/clothing/suit/armor/f13/rangercombat/finlayranger
 	name = "Brotherhood of Steel Ranger Armor"
 	desc = "A customized and well-worn suit of riot gear with parts of the suit reinforced with leather armor."
 	icon_state = "finlayranger"
 	item_state = "finlayranger"
+
+/obj/item/clothing/suit/armor/f13/rangercombat/finlayranger_alt
+	name = "worn ranger armor"
+	desc = "A customized and well-worn suit of riot gear with parts of the suit reinforced with leather armor."
+	icon_state = "reshiranger"
+	item_state = "reshiranger"
+
+/obj/item/clothing/suit/armor/f13/combat/swat/bibledog
+	name = "SLPD SWAT combat armor"
+	slowdown = 0.11
+
+/obj/item/clothing/suit/armor/bulletproof/gelatocoat
+	name = "worn greatcoat"
+	desc = "A heavy bulletproof greatcoat that excels in protecting the wearer against traditional projectile weaponry and explosives to a minor extent."
+	icon_state = "gelatopeacoat"
+	item_state = "gelatopeacoat"
+	strip_delay = 70
+	equip_delay_other = 50
+	slowdown = 0.13
+
+/obj/item/clothing/suit/armor/f13/leather_jacket/combat/reika
```

Comment:
You should make this a subtype of `/obj/item/clothing/suit/armor/f13/leather_jacket/combat/coat` and only change the icon/item states, since those are the only things changed from that definition here.